src/store/reducers/mainForm/index.ts

```diff
@@ -0,0 +1,19 @@
+import { createStore } from "redux";
```

Comment:
create a new file at point /src/store/reducers/index.ts
and use createStore there
This reducer file is specifically for user only, what will you do when there are multiple reducers!